templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/common/PortProxy.ts

```diff
@@ -21,57 +21,56 @@
  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  THE SOFTWARE.
-****************************************************************************/
-import worker from '@ohos.worker';
-import {DedicatedWorkerGlobalScope} from '@ohos.worker';
-// NOTE: don't use this import all statement, because this has compile issue.
-// import * as wk from '@ohos.worker';
+ ****************************************************************************/
+import { ThreadWorkerGlobalScope } from '@ohos.worker';
 import { MessageEvent } from '@ohos.worker';
 
 export class PortProxy {
-    private autoId: number = 0;
-    public actionHandleMap = {}
-    private port :DedicatedWorkerGlobalScope = null;
+  private autoId: number = 0;
+  public actionHandleMap = {}
+  private workerPort: ThreadWorkerGlobalScope = null;
```

Comment:
Why do we need to name it ‘workerPort’ when this interface accepts UIPort and workerPort?